docs/administrators/upgrading_the_orchestrator.rst

```diff
@@ -4,6 +4,53 @@
 Upgrading the orchestrator
 --------------------------
 
+Upgrading the orchestrator can be done either in-place or by setting up a new orchestrator next to the old one
+and migrating the state from the old to the new instance. The sections below describe the upgrade procedure
+for each of both situations.
+
+Upgrading the orchestrator in-place
+###################################
+
+This section describes how to upgrade an orchestrator in-place.
+
+.. note::
+    **Pre-requisite**
+
+    - Before upgrading the orchestrator to a new major version, make sure the old orchestrator is at the latest version available within its major.
```

Comment:
Is the guide about upgrading the orchestrator to a new version or a new major version? In the case of the latter, where is the other update procedure documented?